packages/editor-ui/vite.config.ts

```diff
@@ -60,18 +60,16 @@ export default mergeConfig(
 		},
 		plugins: [
 			vue(),
-			...(!isCI
```

Comment:
@netroy added this to gain a build speed improvement. Does it need to be removed in order for this to work?

Reply:
Yes, it made the tests that were using Monaco extremely flaky. Since we're now caching builds, this ~2mins save doesn't seem as impactful

Reply:
I'm going to wait a bit for sure since I'm also running some tests using the Currents dashboard instead of Cypress. But I don't think the PR should block this PR, as the only related part is this small directive